### PR TITLE
[W5.7][W16-3]Ng Kang Tze

### PR DESCRIPTION
**Pull request by ngkangtze**

### CODE REVIEW

**Comment by nelvintan** [comment]
### Suggestion
Great job. Please consider sorting by other means. For example, you can try sorting by tags.

**Comment by guestuser0123** [comment]
Good job. Do remember to update your IO file to reflect the changes.

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+
```
Comment (joanneong) [inline src/seedu/addressbook/commands/SortCommand.java:7]: Redundant line here :P 

Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

Comment (joanneong) [reply src/seedu/addressbook/commands/SortCommand.java:7]: Same for other areas in your code below!

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
+import java.util.Comparator;
```
Comment (joanneong) [inline src/seedu/addressbook/data/person/UniquePersonList.java:9]: The order of this import statement violates the coding standards. Where should it be instead? 🤔

**Review by joanneong — APPROVED** [review]
Decent attempt at implementing the enhancement :) In future, do update the documentation and tests to reflect changes in the code and functionalities as well!

Please close this PR after reading my comments.

**Pull request closed without merging**